README.md

```diff
@@ -50,6 +50,8 @@ Note that you must also have the __ArcGIS Runtime SDK for iOS__ installed and yo
  4. Drag the `ArcGISToolkit.framework` from the `ArcGISToolkit.xcodeproj/ArcGISToolkit/Products` folder to the "TARGETS" settings for your application and drop it in the "Embedded Binaries" section in the "General" tab
  5. Add `import ArcGISToolkit` in your source code and start using the toolkit components 
 
+## SwiftLint
+New in the 100.6.0 release is SwiftLint support for both the Toolkit and Examples app.  You will need to install SwiftLint from [here](https://github.com/realm/SwiftLint) in order to build.  The specific rules the linter uses can be found in the `swiftlint.yml` files in the `Toolkit` and `Examples` directories.
```

Comment:
> You will need to install SwiftLint… in order to build

The script produces a warning, not an error, when SwiftLint is missing, so users should still be able to build even if they don't have SwiftLint installed.